test/SILOptimizer/generic_specialization_loops_detection_with_loops.swift

```diff
@@ -0,0 +1,105 @@
+// RUN: %target-swift-frontend -O -emit-sil -Xllvm -sil-print-generic-specialization-loops -Xllvm -sil-print-generic-specialization-info %s 2>&1 | %FileCheck --check-prefix=CHECK %s
+
```

Comment:
Yes, that's what I mean. Sorry, I didn't see this check. I think it's good enough.